### PR TITLE
Pr disruption properties

### PR DESCRIPTION
**Pull request by vpassama**
# Description

This PR adds disruption properties in populate_pb in order to send them to navitia.

## Issue (optional)

https://github.com/CanalTP/navitia/pull/2257

## Pull Request type (optional)

feature

## How to test

as usual

## Team reviewers (optional)

Mention here the people who should review your PR

### CODE REVIEW

**Comment by buildbotctp** [comment]
Can one of the admins verify this patch?

File: chaos/populate_pb.py [hunk 1]
```diff
@@ -208,6 +208,16 @@ def populate_cause(cause, cause_pb):
         populate_category(cause.category, cause_pb.category)
 
 
+def populate_property(disruption, disruption_pb):
+    if hasattr(disruption, 'properties'):
```
Comment (kinnou02) [inline chaos/populate_pb.py]: This is required? the Disruption object can't have a `property` fields?

Comment (vpassama) [reply chaos/populate_pb.py]: fixed

**Comment by dvdn** [comment]
For the records,
related to https://github.com/CanalTP/chaos-proto/pull/19

**Comment by pthegner** [comment]
Issue link : CanalTP/navitia#2261

**Pull request merged**